app/src/main/java/com/raulh82vlc/functionalkotlinarrow/data/NetworkToCacheMapper.kt

```diff
@@ -16,20 +16,20 @@
 
 package com.raulh82vlc.functionalkotlinarrow.data
 
-import com.raulh82vlc.functionalkotlinarrow.data.cache.model.FeedItemCacheModel
-import com.raulh82vlc.functionalkotlinarrow.data.network.model.FeedItemApiModel
+import com.raulh82vlc.functionalkotlinarrow.data.cache.model.FeedItemCache
+import com.raulh82vlc.functionalkotlinarrow.data.network.model.FeedItemApi
 import java.util.*
 
     private const val REGEX_WHITESPACE = "\\s+"
 
-    fun map(apiInput: List<FeedItemApiModel>): List<FeedItemCacheModel> {
-        val cacheOutput = ArrayList<FeedItemCacheModel>()
+    fun map(apiInput: List<FeedItemApi>): List<FeedItemCache> {
```

Comment:
Why is this function reimplementing `List#map`?

Reply:
@pakoito Because it's a `*air-quotes*` mapper :trollface: